distributed/utils_test.py

```diff
@@ -423,8 +423,7 @@ def disconnect_all(addresses, timeout=3):
                 not pytest.config.getoption("--runslow"),
                 reason="need --runslow option to run")
 except (AttributeError, ValueError):
-    def slow(*args):
-        pass
+    slow = pytest.mark.slow
```

Comment:
This could cause issues with downstream projects who import distributed and run tests with the `--strict` option. See https://github.com/pandas-dev/pandas/issues/16680. The solution was to register the marker in our `setup.cfg` like https://github.com/pandas-dev/pandas/pull/16797/files